core/domain/voiceover_services.py

```diff
@@ -138,6 +129,37 @@ def get_all_language_accent_codes_for_voiceovers(
     return language_codes_mapping
 
 
+def create_entity_voiceovers_model(
+    entity_voiceovers: voiceover_domain.EntityVoiceovers
+) -> voiceover_models.EntityVoiceoversModel:
+    """Creates an entity voiceovers model instance without putting it into
+    the datastore.
+
+    Args:
+        entity_voiceovers: EntityVoiceovers. An instance of the
+            entity voiceovers domain class.
+
+    Returns:
+        EntityVoiceoversModel. An instance of the entity voiceovers model.
+    """
+    entity_voiceovers_dict = entity_voiceovers.to_dict()
+
+    entity_id = entity_voiceovers_dict['entity_id']
```

Comment:
Per this comment: https://github.com/oppia/oppia/pull/20165#discussion_r1594250376

I see, this is an interesting use case for to_dict().

I think the part I'm confused about is why we can't write `entity_id = entity_voiceovers.id` or similar, and same for a bunch of the others below. I suggest you use the domain object where possible, and use the to_dict() for voiceovers_mapping only.

Reply:
Done